readme.md

```diff
@@ -44,4 +44,4 @@ https://github.com/Zheoni/Hanger-Bot
 
 Which is unfortunately no longer updated :/
 
```

Comment:
You can add yourself as a contributor for adding discord.js v13 support to the module !